docs/editor-material.en-US.md

```diff
@@ -0,0 +1,74 @@
+
+- You can also manually create different types of materials in the resource panel:
+
+![image-20210719175600845](https://gw.alipayobjects.com/zos/OasisHub/01f73e82-8be1-4568-baca-a7de3baf17fb/image-20210719175600845.png)
+
+## Adjust the rendering effect
+
+Generally, the materials imported by glTF have already bound the corresponding textures and parameters, and no operation is required. However, the editor supports secondary processing of materials, and the next step is to adjust the rendering effect for different material types.
+
+### 1. Unlit material
+
+As mentioned above, the Unlit material is not affected by light, you only need to set the baked texture:
+
+![unlit](https://gw.alipayobjects.com/zos/OasisHub/e2639e60-a6ed-416d-9f53-064557261d14/unlit.gif)
+
+Generally, the rendering effect of Unlit is very good, because it sets the high-quality texture obtained by the modeling software through offline rendering. If real-time changes in light are not required, this mode can be used in most scenes.
+
+### 2. BlinnPhong material
+
+BlinnPhong material is affected by the light, generally only need to set the **base color, specular, normal.**
+
+- Base color control main tone:
+
+![baseColor](https://gw.alipayobjects.com/zos/OasisHub/a3ac25e1-36fa-4994-bb24-37b837698478/baseColor.gif)
+
+- The specular reflection controls the color and intensity of the highlight portion:
+
+![specular](https://gw.alipayobjects.com/zos/OasisHub/cbda6aec-63e8-4665-b15e-28adaccd7f19/specular.gif)
+
+- Normal controls the **bump sense** above the vision:
+
+![normal](https://gw.alipayobjects.com/zos/OasisHub/9e48930a-a231-416d-9cb0-7bacc675be0a/normal.gif)
+
+### 3. PBR material
+
+The PBR material follows the conservation of energy and is based on physics. You can also set the basic color, normal and other parameters, but it does not support the control of the reflective color and intensity like the BlinnPhong material, because it does not comply with the conservation of energy, however, the rendering effect can be adjusted more conveniently through the `metallicFactor` and `roughness`, and it can also reflect the surrounding environment. Refer to [IBL mode of ambient light](${docs}light#IBL).
+
+- According to the interaction between light and material in the real world, the insulator, that is, when the `metallicFactor` is 0, the material can also reflect about 4% of pure color light, thereby rendering the surrounding environment, such as the head of the model:
+
+![env](https://gw.alipayobjects.com/zos/OasisHub/c40a665d-9d37-46f1-9206-a29859be75a3/env.gif)
+
+- When we adjust the `metallicFactor` of the material, we can find that the greater the metal, the clearer the surrounding environment, and it starts to change from a pure white color to a colorful one. This is because the dielectric, that is, when the `metallicFactor` is 1, the light will be 100% reflected off the surface of the model, that is, the colorful surrounding environment will be reflected:
```

Comment:
It is better: When the `metallicFactor` of the material is adjusted.

use more passive voice